### ISSUE TITLE
Rats not bringing mined blocks back up to the Rat Quarry block

### ISSUE THREAD

**Issue by ghost** [Alex-the-666/Rats#590]
Minecraft 1.16.5
Forge 36.0.14
Rats 7.1.1
Direwolf20 pack v1.6.1

Placing down a Rat Quarry and supplying 3 rats with a quarry upgrade will cause them to begin digging/mining as expected, however they will not bring mined blocks back up to the Quarry block, but instead will just leave them lay on the ground (where they eventually despawn). I also noticed the following in my server logs:

```
[12Feb2021 22:04:19.934] [Server thread/WARN] [net.minecraft.tileentity.TileEntity/]: Block entity invalid: rats:rat_quarry @ BlockPos{x=199, y=126, z=1655}
[12Feb2021 22:13:23.179] [Server thread/WARN] [net.minecraft.tileentity.TileEntity/]: Block entity invalid: rats:rat_quarry @ BlockPos{x=-60, y=69, z=1864}
```

**Comment by Celophy** [comment]
i have same problem, also the miner

**Comment by GizmoTheMoonPig** [comment]
Fixed in 5e5583927d7cc0727b0c8dbb162f274c0380c77e
